Formula/delve.rb

```diff
@@ -0,0 +1,17 @@
+class Delve < Formula
+  desc "Debugger for the Go programming language"
+  homepage "https://github.com/go-delve/delve"
+  url "https://github.com/go-delve/delve/archive/v1.6.0.tar.gz"
+  sha256 "a10a6fc40d87572c6d3f3becdb1a289269e17526d038749f2fa04dd9f591f26a"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-o", bin/"dlv", "./cmd/dlv"
+  end
+
+  test do
+    assert_match(/^Version: #{version}$/, shell_output("#{bin}/dlv version"))
```

Comment:
```suggestion
    assert_equal "Version: #{version}", shell_output("#{bin}/dlv version").chomp
```

Reply:
This not working, because `dlv version` output like below:

```
Delve Debugger
Version: 1.6.0
Build: $Id: 8cc9751909843dd55a46e8ea2a561544f70db34d $
```

Reply:
You could do
```suggestion
    assert_match "Version: #{version}", shell_output("#{bin}/dlv version").chomp
```

Reply:
I'm fine with the version as-is!